<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>861</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="reviewManage_1">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>30</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>리뷰 관리</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_search">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>1231</width>
       <height>611</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>상가번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>리뷰</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>평점</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="Reviewwirte_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>리뷰 쓰기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>50</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>경기천년바탕 Bold</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>리뷰 조회</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="Review_search_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>리뷰 조회</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Reviewwirte_2">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>40</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>리뷰 쓰기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>60</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>경기천년바탕 Bold</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>리뷰 관리</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_Manage">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>1231</width>
       <height>611</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>상가번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>리뷰</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>평점</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="Review_search_2">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>30</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>리뷰 조회</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reviewManage_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>리뷰 관리</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>50</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>경기천년바탕 Bold</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>리뷰 쓰기</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Review">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>301</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>리뷰</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_score">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>250</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>240</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>평점</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Register">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>490</y>
       <width>281</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string>등록</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
